src/dashboard/alerts/alert.py

```diff
@@ -149,8 +150,8 @@ def alert(self):
         :return:
         """
         # Send email if urgent enough
-        if self.severity > 1:
-            emailsystem.send_email_alert(self)
+        #if self.severity > 1:
```

Comment:
Why is this commented out?

Reply:
Bumping Bri's comment, if the code doesn't work please try and fix and refrain from merging with master.

Reply:
Once the circular import fix from #18 is added in it'll fix this, which hopefully will be merged right after this PR